Apps/W1/Shopify/app/src/Order handling/Tables/ShpfyOrderLineAttribute.Table.al

```diff
@@ -0,0 +1,40 @@
+/// <summary>
```

Comment:
All new Objects require the OSS Header (see other objects)

Reply:
ok, added OSS Header